app/src/main/java/com/vestrel00/daggerbutterknifemvp/ui/common/view/BaseFragment.java

```diff
@@ -47,7 +59,7 @@
  * <b>VIEW BINDING</b>
  * This fragment handles view bind and unbinding.
  */
-public abstract class BaseFragment extends Fragment implements HasFragmentInjector {
+public abstract class BaseFragment extends DialogFragment implements HasFragmentInjector {
```

Comment:
This is the only change required to support `DialogFragment` injection in this project.